src/Microsoft.ML.Core/Data/IEstimator.cs

```diff
@@ -263,7 +264,7 @@ public interface IDataReaderEstimator<in TSource, out TReader>
     /// The transformer is a component that transforms data.
     /// It also supports 'schema propagation' to answer the question of 'how will the data with this schema look, after you transform it?'.
     /// </summary>
-    public interface ITransformer
+    public interface ITransformer : ICanSaveModel
```

Comment:
>ICanSaveModel [](start = 36, length = 13)

We need to reflect this change, and others in our [https://github.com/dotnet/machinelearning/blob/master/docs/code/MlNetHighLevelConcepts.md](https://github.com/dotnet/machinelearning/blob/master/docs/code/MlNetHighLevelConcepts.md)
But that probably required another issue.

Reply:
I am going to publish a separate issue about this, so that maybe we can also double check that it is up to date with everything

---
In reply to: [255254348](https://github.com/dotnet/machinelearning/pull/2431#discussion_r255254348) [](ancestors = 255254348)